api/src/test/java/org/pmiops/workbench/billing/FreeTierBillingServiceTest.java

```diff
@@ -338,29 +330,16 @@ public void checkFreeTierBillingUsage_workspaceMissingCreatorNoNPE() {
   public void maybeSetDollarLimitOverride_true() {
     workbenchConfig.billing.defaultFreeCreditsDollarLimit = 100.0;
     final DbUser user = createUser(SINGLE_WORKSPACE_TEST_USER);
-    assertThat(user.getLastModifiedTime()).isNull();
-
-    // we update the user and should see this last modified time
-    final Instant time2 = START_INSTANT.plusSeconds(1000);
-    CLOCK.setInstant(time2);
 
     assertThat(freeTierBillingService.maybeSetDollarLimitOverride(user, 200.0)).isTrue();
     verify(mockUserServiceAuditor)
         .fireSetFreeTierDollarLimitOverride(user.getUserId(), null, 200.0);
     assertWithinBillingTolerance(freeTierBillingService.getUserFreeTierDollarLimit(user), 200.0);
-    assertThat(userDao.findUserByUserId(user.getUserId()).getLastModifiedTime())
```

Comment:
FYI @jmthibault79 my interpretation is that these were not critical assertions to the test. Let me know if you think this was indirectly trying to detect something else which I should now cover differently.

Reply:
I think the logic here was: we set the lastModified in the service, so let's confirm that we did so.

So it's correct to remove these.